lib/ace/autocomplete.js

```diff
@@ -434,7 +434,14 @@ var FilteredList = function(array, filterText) {
         this.filterText = str;
         matches = this.filterCompletions(matches, this.filterText);
         matches = matches.sort(function(a, b) {
-            return b.exactMatch - a.exactMatch || b.score - a.score;
+            var result = a.exactMatch - b.exactMatch
+            if (result != 0)
+                return result
+            result = a.meta.localeCompare(b.meta)
+            if (result != 0)
+                return result
+            result = b.score - a.score
```

Comment:
This looks like a bug (reverted order comparing to the previous comparisons) unless you know that the score value is reverted as well. Please swap b and a here and set scores to `i` instead of `1000 - i` where you sort the completions